test/access-control.integration.js

```diff
@@ -111,9 +111,15 @@ describe('access control - integration', function() {
           assert.equal(user.password, undefined);
         });
       });
+
+      // user has replaceOnPUT = false; so then both PUT and PATCH should be allowed for update
       lt.describe.whenCalledRemotely('PUT', '/api/users/:id', function() {
```

Comment:
This test will have to be changed when back-ported to 2.x, because `replaceById` will be exposed at `/api/users/:id/replace` only. Could you please add a note so that we don't forget?